src/server/controllers/image.controller.ts

```diff
@@ -220,6 +220,7 @@ export const getInfiniteImagesHandler = async ({
       user: ctx.user,
       headers: { src: 'getInfiniteImagesHandler' },
       include: [...input.include, 'tagIds'],
+      includeBaseModel: true,
```

Comment:
You should use the `include` variable already provided and add a new value there :) Should work the same and be a bit clearner